src/routes/balance.routes.ts

```diff
@@ -51,7 +51,9 @@ export const getAccountsBalances = async (
       try {
         accountsBalances.push({
           account: account.stub,
-          balances: await TradingService.getTradeExecutor(account.exchange)
+          balances: await (
```

Comment:
what's the point of adding this await ?

Reply:
Not sure honestly. Must have been an artifact of some experimenting I was doing. Will remove